pyproject.toml

```diff
@@ -24,7 +24,6 @@ classifiers=[
 dependencies = [
     "numpy",
     "scipy",
```

Comment:
Something we might try after this PR: you can specify the dependencies dynamically with newer `setuptool`. I didn't include it since I wasn't sure how it would interact with PySold's current setuptools version pinning

https://github.com/opera-adt/dolphin/blob/main/pyproject.toml#L23-L34

Reply:
I like this dynamic dependencies setup a lot, it makes a lot of sense.